### PR TITLE
SW-3357 camera logs exeption on camera error and trys over and over again

### PR DESCRIPTION
**Pull request by JFSolution**
fix still showing errors in log but not on frontend

### CODE REVIEW

File: octoprint_mrbeam/iobeam/lid_handler.py [hunk 1]
```diff
@@ -921,6 +921,18 @@ def serve_pictures(
...
             else:
+                if cam.camera_error:
+                    self.stopEvent.set()
+                    self._logger.error(
+                        "Camera error: %s, disabling camera", cam.camera_error
+                    )
+                    self._plugin.user_notification_system.show_notifications(
+                        self._plugin.user_notification_system.get_notification(
+                            "err_cam_conn_err",
+                            err_msg=[str("camera error")],
```
Comment (irlaec) [inline octoprint_mrbeam/iobeam/lid_handler.py]: Is this str() necessary? Or can we just skip it? 
```suggestion
                            err_msg=["camera error"],
```

File: octoprint_mrbeam/iobeam/lid_handler.py [hunk 2]
```diff
@@ -921,6 +921,18 @@ def serve_pictures(
...
+                            err_code="E-01FF-1006",
```
Comment (irlaec) [inline octoprint_mrbeam/iobeam/lid_handler.py]: I think there was a module for the error codes, no?

**Review by irlaec — CHANGES_REQUESTED** [review]

File: octoprint_mrbeam/iobeam/lid_handler.py (hunk 2 above)
Comment (JFSolution) [reply octoprint_mrbeam/iobeam/lid_handler.py]: you are right :)

File: octoprint_mrbeam/iobeam/lid_handler.py (hunk 1 above)
Comment (JFSolution) [reply octoprint_mrbeam/iobeam/lid_handler.py]: ohh I can skip it

**Review by irlaec — APPROVED** [review]

**Pull request merged**